regtests/ww3_tp2.22/input/ww3_ounf.inp

```diff
@@ -0,0 +1,43 @@
+$ -------------------------------------------------------------------- $
+$ WAVEWATCH III Grid output post-processing                            $
+$--------------------------------------------------------------------- $
+$ First output time (yyyymmdd hhmmss), increment of output (s),
+$ and number of output times.
+$
+  19680606 000000 3600. 12
+$
+$ Fields requested --------------------------------------------------- $
+$
+$ Output request flags identifying fields as in ww3_shel.inp. See that
+$ file for a full documentation of field output options. Namelist type
+$ selection is used here (for alternative F/T flags, see ww3_shel.inp).
+$
+ N
+ HS T01 DIR SPR USS USSH
```

Comment:
Hmm, I don't know how I got the mis-match.  I'll adjust it to contain `HS EF T01 DIR SPR USS USSH`.

Reply:
@JessicaMeixner-NOAA I looked into what was going on here.  The mismatch with ww3_shel was already present in `ww3_tp2.2` and we just used that regtest as a base to demonstrate the `USS` and `USSH`.  When EF was added into `ww3_outf.inp` to match `ww3_shel.inp`, ww3_outf crashes because the parameter `EF` has not been implemented in `ww3_outf.F90`.  It would take time to implement and test if this is a direction we want to go.  To me this seems outside of the scope of this PR.  My recommendation would be to just revert back to what it was before.

Reply:
Adding EF to outf is out of the scope of this PR.  We should add an issue about this just for our own record keeping. 
  Does EF work in ounf?  If so, we could just keep it there, remove it from here  assuming it's critical for testing in this PR.  An alternative could be that EF is removed from shel, outf and ounf for consistency.

Reply:
It doesn't look like ounf wrote EF.  Sure, I can make remove it from shel and ounf